cmd/formula-analytics.rb

```diff
@@ -377,7 +379,7 @@ def influx_analytics(args)
         tag = "pkg"
       end
 
-      #TODO: Setup tap filtering if needed
+      # TODO: need tap information for filtering and display full names
```

Comment:
@SMillerDev Need some help here 🙇🏻 ❤️. If you can help me get the `tap_name` and ideally (but not essential) `options` fields in here I can fix the output after that.